src/Meilisearch/Query/SearchQueryBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Meilisearch\Query;
+
+use Meilisearch\Contracts\SearchQuery;
+
+final class SearchQueryBuilder implements SearchQueryBuilderInterface
+{
+    public function build(string $indexName, string $query, array $facets, array $filter): SearchQuery
+    {
+        return (new SearchQuery())
+            ->setIndexUid($indexName)
```

Comment:
Ah, Meilisearch actually calls it 'uid'... Hmm... I always felt that 'index name' was a bit ambiguous, but couldn't think of anything better. Maybe we should rename it in another PR

Reply:
Let's leave the `indexName` for now - we can always unify it later if we wish for 🖖 I don't have any strong opinion on that